tests/e2e/workloads/test_jenkins_simulation.py

```diff
@@ -2,13 +2,21 @@
 import logging
 
 from ocs_ci.framework.testlib import ManageTest, workloads, acceptance
+from ocs_ci.ocs import constants
+from ocs_ci.utility import templating
 
 logger = logging.getLogger(__name__)
 
 
 @pytest.fixture()
-def pod(request, pod_factory, interface_iterate):
-    pod = pod_factory(interface=interface_iterate)
+def pod(request, pvc_factory, pod_factory, interface_iterate):
+    pvc_obj = pvc_factory(interface=interface_iterate, status=constants.STATUS_BOUND,)
+    pod_dict = templating.load_yaml(constants.CSI_CEPHFS_POD_YAML)
+    pod_dict["spec"]["containers"][0][
+        "image"
+    ] = "quay.io/repository/tdesala/git-ubuntu-image"
```

Comment:
Hey @PrasadDesala , can we have this image here:  https://quay.io/organization/ocsci instead of your repository?

Reply:
Done